### ISSUE TITLE
Please add playlist history, so that I can go back / revert a playlist change and forward as well

### ISSUE THREAD

**Issue by tomasts248** [clsid2/mpc-hc#1846]
For example: every time I double click a video it replaces the current playlist which means I'm losing playlist contents that existed before, unless there is way to recover it? At least I'm not aware of it :/
I'm aware of the /add switch but what happens if I drag-drop the video?

**Comment by AllThisThen** [comment]
You can drag-and-drop a file into a MPC-HC player window while holding CTRL to add to playlist without opening the playlist panel.
If you want to keep a playlist going for a long time without saving it frequently, you can shift-click or middle-click the MPC-HC taskbar item to open a new player for viewing other things. When you open a file type associated with MPC-HC, it will play the file in an instance of the program that is currently playing something if one is available or the most recently active MPC-HC window (which you don't want to be the one with your playlist). Managing multiple instances might take some practice.
Also, if you make shortcuts to the media you want to manage, dragging and dropping them into MPC-HC's playlist panel works fine and could be a good way to build and manage larger separate playlists however you like without lots of redundant file copying.

Hope this helps a bit.

**Comment by tomasts248** [comment]
Thanks you.

> you can shift-click or middle-click the MPC-HC taskbar item to open a new player for viewing other things. 

Well I been doing that just without the shortcut but the new video opened replaces the last focused instance of MPHC so is confusing and I made mistakes on this, do note I have coped with this for at least a month and have lost track of how many playlists I already lost. Further if you drag and drop if by mistake i add the video below the playlist it again replaces the playlist, thanks for everything but I thin I will look for another alternative software, data loss is something I'm not willing to stick to.

Perhaps a good alternative solution would be a setting that sets to never replace a current playlist...
